packages/dx-react-grid/src/plugins/table-filter-row.jsx

```diff
@@ -17,10 +21,18 @@ const getFilterTableCellTemplateArgs = (
 ) => ({
   ...params,
   column: params.tableColumn.column,
+  row: params.tableRow.row,
```

Comment:
Why do you pass a row here? It is null for the TableFilterRow and it also is not documented.